package/batocera/emulators/retroarch/libretro/libretro-mrboom/libretro-mrboom.mk

```diff
@@ -0,0 +1,20 @@
+################################################################################
+#
+# MRBOOM
+#
+################################################################################
+# Version.: Commits on Feb 28, 2019
+LIBRETRO_MRBOOM_VERSION = 67b1b65633f67e79e0ff552f4571cbcf0969a535
```

Comment:
change to LIBRETRO_MRBOOM_VERSION = 4.7

Reply:
I think I did it right

Reply:
you did. i only said there is a better way because you are using a stable version.